common/src/main/java/com/wynntils/screens/lists/OverlayList.java

```diff
@@ -54,14 +59,25 @@ public void render(PoseStack poseStack, int mouseX, int mouseY, float partialTic
         OverlayEntry hovered = this.getHovered();
 
         if (hovered != null) {
-            RenderUtils.drawTooltipAt(
-                    poseStack,
-                    mouseX,
-                    mouseY,
-                    100,
-                    HELP_TOOLTIP_LINES,
-                    FontRenderer.getInstance().getFont(),
-                    false);
+            if (!hovered.getOverlay().isParentEnabled()) {
+                RenderUtils.drawTooltipAt(
+                        poseStack,
+                        mouseX,
+                        mouseY,
+                        100,
+                        DISABLED_PARENT_TOOLTIP_LINES,
+                        FontRenderer.getInstance().getFont(),
+                        false);
```

Comment:
A list of features to enable would be helpful for the use I think.

Reply:
I will add the feature to the tooltip.

Reply:
Done